mendoza/init.janet

```diff
@@ -185,23 +185,19 @@
     # watch runs on linux and mac at least
     # detection is not a solution for windows
     # fswatch technically can run on windows however
-    (def proc (file/popen (string "which " command)))
-    (if (not proc) false
-      (let [result (:read proc :line)]
-        (file/close proc)
-        (and result (not (string/has-suffix? "not found\n" result))))))
+    (def proc (os/spawn ["which" command] :p {:out :pipe}))
+    (= (:wait proc) 0))
```

Comment:
```suggestion
(zero? (:wait proc))
```

Reply:
Yep, I have seen @sogaiu suggesting it, so I would go with them :-)